isoly.ts

```diff
@@ -0,0 +1,29 @@
+import { CallingCode } from "./CallingCode"
```

Comment:
I think all of these ca be changed to `export` directly. (And removing export-block.)